tests/mocks/types_module_module.go

```diff
@@ -6,7 +6,7 @@ package mocks
 
```

Comment:
This file is autogenerated by mockgen. Revert, update the Makefile and regenerate this file please.

Reply:
Thanks, updated

Reply:
This is not yet resolved

Reply:
Updated now, I had it in my local previously